src/DocumentFormat.OpenXml/OpenXmlReader.cs

```diff
@@ -241,6 +241,47 @@ public virtual bool HasAttributes
         /// <remarks>The new current element is the end of the element after LoadCurrentElement().</remarks>
         public abstract OpenXmlElement LoadCurrentElement();
 
+        /// <summary>
+        /// Loads the element at current cursor as the specified type.
+        /// </summary>
+        /// <typeparam name="T">The concrete type of the OpenXmlElement to load.</typeparam>
+        /// <returns>The OpenXmlElement that was loaded.</returns>
+        /// <exception cref="InvalidOperationException">Thrown when the current is the end element or when the ElementType and <typeparamref name="T"/> do not match.</exception>
+        public virtual T LoadElement<T>()
+            where T : OpenXmlElement
+        {
+            if (ElementType == typeof(T))
+            {
+                return (T)LoadCurrentElement();
+            }
+            else
+            {
+                throw new InvalidOperationException(ExceptionMessages.ReaderIncorrectTypeOnLoad);
```

Comment:
The message has format strings, but doesn't look like you format it. Use `string.Format` to format it